app/assets/stylesheets/components/front/c-cover.scss

```diff
@@ -0,0 +1,57 @@
+
+.c-cover {
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: center;
+  height: 500px;
+  // image sample. This property should be injected through javascript
+  background-image: asset-url('forest.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  z-index: -1;
+
+  &::after {
+    display: inline-block;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+
+    @if $theme == 2 {
+      background-color: rgba($color-5, .3);
+    } @else {
+      background-color: rgba($color-1, .4);
+    }
+
+    content: '';
```

Comment:
Maybe you could group these two properties with the other instead of putting them after the condition.